### ISSUE TITLE
Startup warning about unsupported class will impact performance

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#13625]
When starting Metabase since 0.37, a warning is shown right at the beginning:
```
$ java -DMB_JETTY_PORT=3001 -jar metabase.jar
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
2020-10-29 15:01:39,112 INFO metabase.util :: Maximum memory available to JVM: 1.9 GB
....
```

It comes from the upgrade to Log4j2, and it has no effect on Metabase performance, but it might cause confusion to as if Metabase operation will be slower.
https://stackoverflow.com/questions/53049346/is-log4j2-compatible-with-java-11

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by robdaemon** [comment]
Looks like we can solve this with a manifest entry - that's pretty straightforward.

**Comment by jeff-bruemmer** [comment]
Just adding that, while this issue doesn't affect performance, this warning seems to cause a lot of confusion, so even though it's only a P3, we might want to tackle it sooner rather than later, especially if it's just adding a manifest entry.

**Comment by jeff303** [comment]
Reopening issue, because we had to back out the change due to a performance regression (see [here](https://github.com/metabase/metabase/issues/16380#issuecomment-878731691)).

**Comment by flamber** [comment]
Fixed by #24366

**Comment by nodesocket** [comment]
I'm seeing this on Metabase `0.44.6`.

```bash
Warning: protocol #'java-time.core/Amount is overwriting function abs
WARNING: abs already refers to: #'clojure.core/abs in namespace: java-time.core, being replaced by: #'java-time.core/abs
WARNING: abs already refers to: #'clojure.core/abs in namespace: java-time, being replaced by: #'java-time/abs
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
2022-11-29 17:55:54,375 INFO metabase.util :: Maximum memory available to JVM: 1.9 GB
```

Is this still expected?

**Comment by flamber** [comment]
@nodesocket Yes, this was fixed in 0.45.0 as you can see from the Milestone on this issue.
